manifests/d/DWANGO/SeirenVoice/yukari/complateV2_1.4.0-2.0.0/DWANGO.SeirenVoice.yukari.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DWANGO.SeirenVoice.yukari
+PackageVersion: StandardV2_1.4.0-2.0.0

```

Comment:
Please update this file's path to match the `PackageVersion`:
- actual:`manifests\d\DWANGO\SeirenVoice\yukari\`**complate**`V2_1.4.0-2.0.0` 
- goal: `manifests\d\DWANGO\SeirenVoice\yukari\`**Standard**`V2_1.4.0-2.0.0`